server/src/main/java/org/apache/iotdb/db/mpp/execution/QueryExecution.java

```diff
@@ -204,7 +221,7 @@ public ExecutionResult getStatus() {
     }
   }
 
-  private synchronized void initialResultHandle() throws IOException {
+  private void initialResultHandle() throws IOException {
```

Comment:
initial -> init or initialize.

initial means signing (a document) with one's initials in order to authorize or validate it.

Reply:
fix